### PR TITLE
DOCSP 33761 - adding to C2C quickstart

### PR DESCRIPTION
**Pull request by ltran-mdb2**
## DESCRIPTION

adding info on one time sync and continuous replication on the C2C quickstart page

## STAGING

https://preview-mongodbltranmdb2.gatsbyjs.io/cluster-sync/DOCSP-33761/quickstart/#synchronize-data-between-clusters

## JIRA

https://jira.mongodb.org/browse/DOCSP-33761

## BUILD LOG

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=656758f27a5cb5fef530c914

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Is this free of spelling errors?
- [x] Is this free of grammatical errors?
- [x] Is this free of staging / rendering issues?
- [x] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/quickstart.txt [hunk 1]
```diff
@@ -219,6 +220,34 @@ The API documentation provides details on using the following endpoints:
...
+-------------
+
+For a one-time sync, after initializing data synchronization, use the
+:ref:`progress <c2c-api-progress>` command to see the status of the
+synchronization process: 
+
+.. literalinclude:: /includes/api/requests/progress.sh
+   :language: shell
+
+The following requirements must be met for a one-time sync:
```
Comment (ajhuh-mdb) [inline source/quickstart.txt:233]: [suggestion] active voice - I think we can reword this to be more active 

Maybe something along the lines of: 
"For a one-time sync, verify that the ``progress`` response includes the following field values: "

File: source/quickstart.txt [hunk 2]
```diff
@@ -219,6 +220,34 @@ The API documentation provides details on using the following endpoints:
    * - :ref:`c2c-api-reverse`
      - Reverses the direction of a committed sync operation.
 
+One-Time Sync
+-------------
+
+For a one-time sync, after initializing data synchronization, use the
+:ref:`progress <c2c-api-progress>` command to see the status of the
```
Comment (ajhuh-mdb) [inline source/quickstart.txt:227]: [nit] Progress is an endpoint, not a command, so we should reword this sentence to avoid confusing users. Typically, we refer endpoints by "calling" them (i.e. "after initializing data synchronization, **call** the :ref:`progress <c2c-api-progress>` endpoint...")

File: source/quickstart.txt [hunk 3]
```diff
@@ -219,6 +220,34 @@ The API documentation provides details on using the following endpoints:
...
+.. literalinclude:: /includes/api/requests/progress.sh
+   :language: shell
+
+The following requirements must be met for a one-time sync: 
+
+  - ``state: "RUNNING"``
+  - ``canCommit: true``
+  - ``lagTimeSeconds`` is near ``0`` (*Recommended, but not required*)
+
+Then, use the :ref:`commit <c2c-api-commit>` command to commit the
```
Comment (ajhuh-mdb) [inline source/quickstart.txt:239]: Same comment as above with `progress` about referring to endpoints!

File: source/quickstart.txt [hunk 4]
```diff
@@ -219,6 +220,34 @@ The API documentation provides details on using the following endpoints:
...
+synchronization operation to the destination cluster and stop continuous
+replication:
+
+.. literalinclude:: /includes/api/requests/commit.sh
+   :language: shell
+
+If the :ref:`commit <c2c-api-commit>` request is sucessful,
```
Comment (ajhuh-mdb) [inline source/quickstart.txt:246]: [nit] typo + [minimize repeated links](https://www.mongodb.com/docs/meta/style-guide/style/links-and-cross-references/#minimize-repeated-links)
```suggestion
If the ``commit`` request is successful,
```

File: source/quickstart.txt [hunk 5]
```diff
@@ -219,6 +220,34 @@ The API documentation provides details on using the following endpoints:
...
+``mongosync`` returns ``{"success":true}`` and enters the ``COMMITTING``
```
Comment (ajhuh-mdb) [inline source/quickstart.txt:247]: text formatting for readability
```suggestion
``mongosync`` returns ``{ "success": true }`` and enters the ``COMMITTING``
```

File: source/quickstart.txt [hunk 6]
```diff
@@ -219,6 +220,34 @@ The API documentation provides details on using the following endpoints:
...
+state. After the sync is complete, ``mongosync`` is in the ``COMMITTED``
```
Comment (ajhuh-mdb) [inline source/quickstart.txt:248]: [suggestion] wording for active voice
```suggestion
state. After the sync is complete, ``mongosync``enters the ``COMMITTED``
```

File: source/quickstart.txt [hunk 7]
```diff
@@ -180,22 +180,23 @@ Initialization Notes
...
    :language: shell
 
-Once started, ``mongosync`` will synchronize existing data on the
-source cluster with the destination cluster. After the initial sync has
-completed, new writes to the source cluster will be synced with the
-destination cluster.
+If the :ref:`start <c2c-api-start>` request is successful, ``mongosync``
+returns ``{"success":true}`` and starts to synchronize existing data on
+the source cluster with the destination cluster. At this point,
+``mongosync`` is in the ``RUNNING`` state and applies subsequent source
```
Comment (ajhuh-mdb) [inline source/quickstart.txt:193]: [suggestion] for active voice
```suggestion
``mongosync`` enters the ``RUNNING`` state and applies subsequent source
```

File: source/quickstart.txt [hunk 8]
```diff
@@ -219,6 +220,34 @@ The API documentation provides details on using the following endpoints:
    * - :ref:`c2c-api-reverse`
      - Reverses the direction of a committed sync operation.
 
+One-Time Sync
+-------------
```
Comment (ajhuh-mdb) [inline source/quickstart.txt:224]: [question] Should this be a H2 header? It looks like this information is pertinent to the "Synchronize Data Between Clusters" content above. If you feel this should stay as an H1, should we change and/or move the "Synchronizations Notes" section/header that's currently at the bottom of the page?

**Review by ajhuh-mdb — CHANGES_REQUESTED** [review]
Thanks for making these updates @ltran-mdb2 ! I left some comments and suggestion for you to consider. Feel free to ping me if anything's unclear.

**Comment by ltran-mdb2** [comment]
Thanks for your feedback @ajhuh-mdb ! Could you take another look when you get a chance?

**Review by ajhuh-mdb — APPROVED** [review]
LGTM! Thanks for making these updates!
